src/custom/state/price/updater.ts

```diff
@@ -161,19 +160,13 @@ export default function FeesUpdater(): null {
       const refetchPrice = !unsupportedToken && priceIsOld(quoteInfo)
 
       if (unsupportedNeedsCheck || refetchAll || refetchPrice) {
-        const shouldFetchNewQuote = quoteInfo && !quoteUsingSameParameters(quoteParams, quoteInfo)
+        const shouldFetchNewQuote = (quoteInfo && !quoteUsingSameParameters(quoteParams, quoteInfo)) || true
```

Comment:
so... this is always true..

Reply:
i'm concerned about this @anxolin

Reply:
mmm great catch. Let me review this!!!

Reply:
mmm, not sure but in which PR but this code was changed short after: https://github.com/gnosis/cowswap/blob/548b5e29214c35f629910cbe74015621dd2e5225/src/custom/state/price/updater.ts#L178-L179 

So I guess this is solved.